docs/src/documentation/03-components/01-action/buttonmobilestore/03-accessibility.mdx

```diff
@@ -0,0 +1,75 @@
+  - /components/buttonmobilestore/accessibility/
+---
+
+## Accessibility
+
+The ButtonMobileStore component has been designed with accessibility in mind, providing accessible mobile app store download buttons with proper semantic structure and screen reader support.
+
+### Accessibility Props
+
+The following props are available to improve the accessibility of your ButtonMobileStore component:
+
+| Name            | Type      | Description                                                                                                                                                                                                                             |
+| :-------------- | :-------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| alt             | `string`  | **Required.** Provides alternative text description of the store button image for screen readers. Should clearly describe the action and store type (e.g., "Download on the App Store" or "Get it on Google Play"). Must be translated. |
+| title           | `string`  | Provides additional context as a tooltip and is announced by screen readers after the alt text. Use to provide supplementary information about the app or download. Must be translated.                                                 |
+| stopPropagation | `boolean` | Controls event bubbling behavior. When `true`, prevents the click event from bubbling up to parent elements, which can help avoid focus management issues in complex layouts.                                                           |
+
+### Automatic Accessibility Features
+
+- The component automatically renders as a semantic `<a>` element, creating a proper link that assistive technologies can identify and navigate
+- The component uses an `<img>` element with proper `srcSet` for responsive images while maintaining accessibility through the required `alt` attribute
+- Focus management follows standard link behavior with visible focus indicators
+- The component structure ensures screen readers announce both the alt text and any title information
+
+### Best Practices
+
+- Always provide a descriptive and translated `alt` text that clearly indicates the action and store type
+- Use specific alt text like "Download on the App Store" or "Get it on Google Play" rather than generic text like "Download app"
+- Ensure the `href` points to the actual app store page for the specific app to provide a functional download experience
+- When using the `title` prop, provide additional translated context that supplements but doesn't duplicate the alt text.
+- Consider the language of your users and use the appropriate `lang` prop to display store buttons in the correct language
+- Ensure sufficient color contrast between the button and its background for users with visual impairments
```

Comment:
In `TextLink` [we added recommendation to inform users that the link will be opened in a new tab (when true).](https://orbit.kiwi/components/action/textlink/accessibility/#best-practices) In this component, link will always open in a new tab, should we also handle this here?